frontend/src/components/Interactions/Edit.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <button
+        class="material-icons edit-button"
+        type="menu"
+        title="Toggle Menu"
+    >
+        more_vert
```

Comment:
Doesn't this make more sense as a menu which contains stuff like 'Edit', 'Delete' etc?

Reply:
I think more_vert is okay, but this file shouldn't be called edit.

Reply:
yes